.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches: [ dev ]
+  workflow_dispatch:
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: npm install && npm run build
+        env:
+          GITHUB_REPO_NAME: ${{ github.event.repository.name }}
+          VITE_GITHUB_REPO_NAME: /${{ github.event.repository.name }}
```

Comment:
Are you sure you need both ENV variables? Can you explain why?

Reply:
Not fixed. Remove `GITHUB_REPO_NAME` if you don't use it.

Reply:
Simplified. Using a singe ENV instead of two https://github.com/Adamant-im/adamant-im/pull/745/commits/a4325d695e81b379d3f3968f6992faa0e7a5402d